packages/viewer/src/lib/interactions/interactionApi.ts

```diff
@@ -370,25 +370,25 @@ export abstract class InteractionApi<T extends Camera = Camera> {
    *  viewer.
    */
   public async rotateCamera(delta: Point.Point): Promise<void> {
-    return this.transformCamera(({ camera, viewport }) => {
+    return this.transformCamera(({ camera, viewport, boundingBox }) => {
       const upVector = Vector3.normalize(camera.up);
-      const lookAt = Vector3.normalize(
+      const directionVector = Vector3.normalize(
```

Comment:
No, _directionVector_ is consistent because it's normalized. In our code, `viewVector = Vector3.subtract(camera.lookAt, camera.position)` and the direction vector is the normalized viewVector.